server/graphql_schemas/utils/helpers.py

```diff
@@ -213,5 +213,24 @@ def add_event_to_calendar(andela_user, event):
                  for attendee in event.attendees]
     calendar = build('calendar', 'v3', credentials=andela_user.credential)
     event_details = build_event(event, attendees)
-    return calendar.events().insert(
+    created_event = calendar.events().insert(
             calendarId='primary', body=event_details).execute()
+    event.event_id_in_calendar = created_event['id']
+    event.save()
+
+
+def update_event_status_on_calendar(andela_user, event):
+    event_id = event.event_id_in_calendar
+    attendee = andela_user.user.email
+    calendar = build('calendar', 'v3', credentials=andela_user.credential)
+    event_in_calendar = calendar.events().get(calendarId='primary', eventId=event_id).execute()
+    attendees = event_in_calendar['attendees']
+    for attendee in attendees:
+        if attendee['email'] == andela_user.user.email:
+            attendee['responseStatus'] = 'accepted'
```

Comment:
It seems this only works for users who have already received invites for the event, due to the fact that they were sent invites when an event was created in their category of interest. 
If a user tries to attend an event which they were not sent invites for, due to the fact that it wasn't created under their category of interest, this implementation won't work. 
Kindly help address this. Thanks

Reply:
I have fixed this